src/components/SpiceList.js

```diff
@@ -8,19 +8,36 @@ class SpiceList extends React.Component {
     search: ""
   }
 
+  getSearchTerm = (searchTerm) => {
+    this.setState({
+      search: searchTerm
+    })
+  }
+
+  toggleFilter = (toggleStatus) => {
+    this.setState({
+      fourStarOnly: toggleStatus
+    })
+  }
+
   renderSpices() {
     return this.props.spices
       .filter(spice => this.state.fourStarOnly ? spice.rating >= 4 : true)
       .filter(spice => spice.notes.toLowerCase().includes(this.state.search.toLowerCase()))
-      .map(spice => (
-        <SpiceItem key={spice.id} spice={spice} />
+      .map((spice, index) => (
+        <SpiceItem key={index} spice={spice} />
       ))
```

Comment:
In general it's [not recommended to use the index as the `key` prop](https://reactjs.org/docs/lists-and-keys.html#keys) for any list that will be sorted/reorganized, since it can cause bugs. Since the newly added spices don't have an ID, you could use a different attribute such as the title (so long as it's unique).